src/commons/utils/JavaHelper.ts

```diff
@@ -0,0 +1,145 @@
+import setupJVM, { parseBin } from 'java-slang/dist/jvm';
+import { createModuleProxy, loadCachedFiles } from 'java-slang/dist/jvm/utils/integration';
+import { Context } from 'js-slang';
+import { initModuleContext, loadModuleBundle } from 'js-slang/dist/modules/loader/moduleLoader';
+
+import DisplayBufferService from './DisplayBufferService';
+
+const CDN = 'https://source-academy.github.io/modules/java/java-packages/src/';
```

Comment:
I think best to move this to the `Constants` object in `Constants.ts`?

Reply:
updated.